content/docs/start/data/metrics-parameters-plots.md

```diff
@@ -0,0 +1,278 @@
+---
+title: 'Get Started: Metrics, Parameters, and Plots'
```

Comment:
QQ - is it the same file as we had before or do we have some / any changes here?

Reply:
Same file. I did minimum content changes in this PR